### ISSUE TITLE
Installation fails

### ISSUE THREAD

**Issue by thegunshow** [urbiworx/node-red-contrib-freeboard#22]
Hi

This node is failing during installation.

Using the recommended versions of NPM and Node-Red.

Anyone else have the same problem?

[npm.txt](https://github.com/urbiworx/node-red-contrib-freeboard/files/376321/npm.txt)

Thanks

**Comment by mvherweg** [comment]
Install also failing here (on OS X) for version 0.0.7 (0.0.6 works fine)

```
$ npm install node-red-contrib-freeboard

> node-red-contrib-freeboard@0.0.7 postinstall /Users/mvherweg/.node-red/node_modules/node-red-contrib-freeboard
> node rewritefiles.js

{ Error: ENOENT: no such file or directory, open 'node_modules/freeboard/index.html'
    at Error (native)
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'node_modules/freeboard/index.html' }
{ Error: ENOENT: no such file or directory, open 'node_modules/freeboard/js/freeboard.js'
    at Error (native)
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'node_modules/freeboard/js/freeboard.js' }
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open 'node_modules/freeboard/plugins/thirdparty/jquery.keyframes.min.js'
    at Error (native)
```

**Comment by marclar** [comment]
Me too; only 0.0.5 worked for me.

**Comment by casaria** [comment]
Me too, tried 2 different node-red installation servers -> fails the same way during install

**Comment by timonsku** [comment]
The same here on Raspberry Pi on Raspbian.

**Comment by av1024** [comment]
Confirm.
Can be installed manually:
 0) cd ~/.node-red
 1) git clone https://github.com/Freeboard/freeboard.git
 2) npm install freeboard
 3) git clone https://github.com/urbiworx/node-red-contrib-freeboard.git
 4) npm install node-red-contrib-freeboard

Looks like install rule for freeboard does not work

But "save" button silently does not work for me.
Tested on ubuntu/Mint x84_64 with "clean" nodejs 6.x and latest "clean" node-red installation

UPD: there is js error on freeboard.js: saveDasboardClicked: `event` is undefined here, looks like freeboard issue

**Comment by jsilence** [comment]
manual installation does not work for me. Steps complete fine, but the freeboard node does not show up in the node list.

**Comment by MehrCurry** [comment]
+1

**Comment by kuedan** [comment]
This is the message i get from either manual or normal install:
npm WARN .node-red Invalid dependency: zeparser undefined

npm ERR! code Z_BUF_ERROR
npm ERR! errno -5
npm ERR! unexpected end of file

**Comment by netoolii** [comment]
same issue here. (Ubuntu 16)


```
> node-red-contrib-freeboard@0.0.7 postinstall /usr/lib/node_modules/node-red-contrib-freeboard
> node rewritefiles.js

events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: EACCES: permission denied, open 'node_modules/freeboard/plugins/thirdparty/jquery.keyframes.min.js'
    at Error (native)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-red-contrib-freeboard@0.0.7 postinstall: `node rewritefiles.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-red-contrib-freeboard@0.0.7 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2017-07-17T19_20_04_459Z-debug.log
```

**Comment by dolfandringa** [comment]
Wow, this is over a year old already and still not solved?
I got the problem too.

**Comment by vleud101** [comment]
#27 

Same here! I hope there is a fix soon, because i need it for lecture 7 on noderedguide.com :)

**Comment by jsilence** [comment]
@vleud101, I guess there will be no fix.

Newer versions of NodeRed offer a dashboard toolkit that is capable to replace at least part of the functionality of FreeBoard, if not all:

https://flows.nodered.org/node/node-red-dashboard

**Comment by vleud101** [comment]
Hi @jsilence  thanks for your comment so quickly, I did knwo there was a kind of that stuf but i hoped to learn it with the tuturiols : )  (http://noderedguide.com/lecture-7-node-red-dashboard-part1/) 

Thanks again, I gonna take a look!

**Comment by dolfandringa** [comment]
Yeah, node-red-dashboard is very nice to be honest. It is not a solution to this issue, but I am using it too in favor of freeboard. So I don't need freeboard.

**Comment by rahul2605** [comment]
I was having the same issue but it works if I do "npm install freeboard" and then "npm install node-red-contrib-freeboard".

**Comment by PatLub85** [comment]
I have the same problem with node v12.14.1, npm  v6.13.4 and the message : Error while executing:
 undefined ls-remote -h -t git://github.com/freeboard/freeboard.git ???
Do you have an idea?
Thanks
